modules/ROOT/pages/manage-dependency-versions-design-center.adoc

```diff
@@ -10,6 +10,15 @@ Dependencies are connectors and other parts of a flow that are versioned indepen
 
 Switching will not cause your project to delete the reusable configuration that you have created for the connector. However, if the changes that are introduced in the version that you switch to include configuration changes, you might need to edit the configuration for the connector. To check whether editing is necessary, refer to the release notes for the connector.
 
+[IMPORTANT]
+====
+After a new version of Mule runtime is released, opening an existing Mule-app project causes the application to use that new version. The application is updated immediately, Flow Designer then restarting it in the Design environment only. If Flow Designer is unable to update an application, the application is not restarted.
+
+This process ensures that Mule applications are always using the latest version of Mule Runtime.
```

Comment:
-Mule runtime.

Reply:
I think you meant "Mule Runtime". Fixed.